# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextWGL.h
# Commit: bbfe23c61add
# Full Hash: bbfe23c61add07377daf80696fd6996c0463f5b6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-09 09:21:34
# Regressor Bug: 1632249
# File Overlap Count: 1
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 8fcfac450ecc -r bbfe23c61add gfx/gl/GLContextWGL.h
--- a/gfx/gl/GLContextWGL.h	Mon Jun 08 20:13:01 2020 +0000
+++ b/gfx/gl/GLContextWGL.h	Mon Jun 08 20:34:15 2020 +0000
@@ -17,13 +17,11 @@
  public:
   MOZ_DECLARE_REFCOUNTED_VIRTUAL_TYPENAME(GLContextWGL, override)
   // From Window: (possibly for offscreen!)
-  GLContextWGL(CreateContextFlags flags, const SurfaceCaps& caps,
-               bool isOffscreen, HDC aDC, HGLRC aContext,
+  GLContextWGL(const GLContextDesc&, HDC aDC, HGLRC aContext,
                HWND aWindow = nullptr);
 
   // From PBuffer
-  GLContextWGL(CreateContextFlags flags, const SurfaceCaps& caps,
-               bool isOffscreen, HANDLE aPbuffer, HDC aDC, HGLRC aContext,
+  GLContextWGL(const GLContextDesc&, HANDLE aPbuffer, HDC aDC, HGLRC aContext,
                int aPixelFormat);
 
   ~GLContextWGL();